File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    w,h = [],[]
    for i in sizes:
        if i[0]<i[1]:
            w.append(i[1])
            h.append(i[0])
        else:
            w.append(i[0])
            h.append(i[1])
            
    answer = max(w)*max(h)
    return answer

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        int x=0; int y=0;
        for(int[] s: sizes) {
            int tmpX = Math.max(s[0], s[1]);
            int tmpY = Math.min(s[0], s[1]);
            
            x = Math.max(x, tmpX);
            y = Math.max(y, tmpY);
        }
        
        return x*y;
    }
}